package.json

```diff
@@ -2327,6 +2337,11 @@
 					"group": "inline@2",
 					"when": "view == notifications:github && (viewItem == 'Issue' || viewItem == 'PullRequest')"
 				},
+				{
+					"command": "notification.markAsRead",
+					"group": "inline@3",
+					"when": "view == notifications:github"
```

Comment:
I guess we don't need a "is unread" in this `when` because we don't show read notifications?

Reply:
That's right, currently we only show unread notifications.

Reply:
Perhaps it would make sense to show also read notifications at some point 🤔